eslint/src/core/check-version.ts

```diff
@@ -0,0 +1,26 @@
+import {lt} from 'semver';
+import color from 'kleur';
+
+const eslintPackageName = '@exadel/eslint-plugin-esl';
+const eslPackageName = '@exadel/esl';
```

Comment:
Cosmetic, constant notation
```suggestion
const ESL_PACKAGE_NAME = '@exadel/esl';
const PLUGIN_PACKAGE_NAME = '@exadel/eslint-plugin-esl';
```
or just
```suggestion
const ESL_PACKAGE = '@exadel/esl';
const PLUGIN_PACKAGE = '@exadel/eslint-plugin-esl';
```